internal/workflow/processing_test.go

```diff
@@ -68,8 +78,9 @@ func (s *ProcessingWorkflowTestSuite) SetupWorkflowTest(taskQueue string) {
 	ctrl := gomock.NewController(s.T())
 	logger := logr.Discard()
 	cfg := config.Configuration{
-		Preservation: pres.Config{TaskQueue: taskQueue},
-		A3m:          a3m.Config{ShareDir: s.transferDir},
+		Preservation:  pres.Config{TaskQueue: taskQueue},
+		A3m:           a3m.Config{ShareDir: s.transferDir},
+		Preprocessing: prepConfig,
```

Comment:
Nitpick: this is really minor, but "prep" is usually a short form for "prepare" so I read "prepConfig" as "prepare config".  Maybe "ppConfig" or "preprocessingConfig" instead?